src/AspNet.Security.OAuth.EVEOnline/EVEOnlineAuthenticationConstants.cs

```diff
@@ -1,19 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
- * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
- * for more information concerning the license and the contributors participating to this project.
- */
-
-namespace AspNet.Security.OAuth.EVEOnline
-{
-    /// <summary>
-    /// Contains constants specific to the <see cref="EVEOnlineAuthenticationHandler"/>.
-    /// </summary>
-    public static class EVEOnlineAuthenticationConstants
```

Comment:
Please restore this file and mark it as `[Obsolete]` with an appropriate message instead. Otherwise this is a binary breaking change.

Reply:
I have re-added this and also added the code to add the scopes to the claim which I missed.